plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/model/SQLServerTableColumn.java

```diff
@@ -240,7 +236,11 @@ public boolean isRequired() {
     @Override
     public boolean isAutoGenerated() {
         // Computed columns are "auto-generated" (from supplied definition)
-        return super.isAutoGenerated() || CommonUtils.isNotEmpty(getComputedDefinition());
+        return super.isAutoGenerated() || isTimestamp() || CommonUtils.isNotEmpty(getComputedDefinition());
+    }
+
+    public boolean isTimestamp() {
+        return dataType.getName().equals(SQLServerConstants.TYPE_TIMESTAMP);
```

Comment:
If I understand the provided DDL correctly, the type of the column is datetime, not timestamp. Probably, we should do it for all datetime types.